torch/nn/quantized/modules/conv.py

```diff
@@ -481,9 +491,10 @@ def from_float(cls, mod):
             mod (Module): a float module, either produced by torch.quantization
               utilities or provided by the user
         """
-        assert type(mod) == cls._FLOAT_MODULE, \
-            ' nnq.' + cls.__name__ + '.from_float only works for ' + \
-            cls._FLOAT_MODULE.__name__
+        # derived classes override cls._FLOAT_MODULE attribute
+        msg = ' nnq.' + cls.__name__ + '.from_float only works for ' + \
+              cls._FLOAT_MODULE.__name__  # type: ignore[attr-defined]
```

Comment:
ignores could be avoided by:
```
MOD = TypeVar('MOD', bound=nn.modules.conv._ConvNd)

class _ConvNd(nn.Module):

    # Should be redefined by each subclass
    _FLOAT_MODULE = MOD
```
I think. Again this code will work with the ignore in practice, so I'm not sure the change should be made.